dojo/tools/meterian/parser.py

```diff
@@ -0,0 +1,139 @@
+import json
+from urllib.parse import urlparse
+
+from dojo.models import Finding
+
+
+class MeterianParser(object):
+
+    def get_scan_types(self):
+        return ["Meterian Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Meterian JSON report output file can be imported."
+
+    def get_findings(self, report, test):
+        findings = []
+        report_json = self.parse_json(report)
+
+        security_reports = report_json["reports"]["security"]["reports"]
```

Comment:
This could break. We saw a lot of issues due to keys not present in the files.
Could you add some checks that raise `ValueError("...")` if the file is invalid (with unit tests to check that)?

Reply:
Now checking whether the report file is not malformed raising the a `ValueError` exception as suggested.